FilteringApp/FilteringApp.Core/Utils/ArrayUtils.cs

```diff
@@ -39,38 +40,55 @@ public static int AddArrayElements(int[] array)
             return theSum;
         }
 
-        public static ArrayParseResult CheckArray(string[] stringArray)
+        public static int[] CheckArray(this string[] stringArray)
         {
             var array = new int[stringArray.Length];
+
             for (int i = 0; i < array.Length; i++)
             {
                 if (!int.TryParse(stringArray[i], out var value))
                 {
-                    var error = $"Wrong Input in array: {stringArray[i]} Postion :{i + 1}";
-                    return new ArrayParseResult(error);
-                }
-                else
-                {
-                    array[i] = value;
+                    throw new ArgumentException($"Wrong Input in array: {stringArray[i]} Postion :{i + 1}", "ArgumentException");
                 }
-            }
 
-            return new ArrayParseResult(array);
+                array[i] = value;               
+            }
+            return array;
         }
 
-        public static UserValueParseResult CheckFilteringValue(string value)
+        public static void Foo(int? i)
```

Comment:
Leftover from one of our lections, should be removed

Reply:
Removed.